### PR TITLE
Add persistant comments via datastore

### PR DESCRIPTION
**Pull request by MrBanananator**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 1]
```diff
@@ -14,13 +14,12 @@
...
+public final class Comment {
+    private final String comment;
+    private final Long timestamp;
 
-public class Comments {
-    private final List<String> comments = new ArrayList<>();
-
-    public void addComment(String comment) {
-        comments.add(comment);
+    public Comment (String comment, Long timestamp) {
```
Comment (Marine5174-zz) [inline portfolio/src/main/java/com/google/sps/data/Comment.java]: I would recommend documenting the unit of the timestamp, such as "timestampMillis" or "timestampMs" or "timestampMicros".

Another possible solution is using java.time.Instant. Not sure how client friendly it is, though.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -26,23 +26,52 @@
...
+
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery results = datastore.prepare(query);
+
+        List<Comment> comments = new ArrayList<>();
+        for (Entity entity : results.asIterable()) {
+            String comment = (String) entity.getProperty("comment");
+            long timestamp = (long) entity.getProperty("timestamp");
+
+            Comment nextComment = new Comment(comment, timestamp);
```
Comment (Marine5174-zz) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Prefer to use clearer names such as:

String content = (String) entity.getProperty("comment");
long timestamp = ...

Comment comment = new Comment(content, timestmap);
comments.add(comment);

**Review by Marine5174-zz — APPROVED** [review]
Nice work! Please address the comments above before merging.

File: portfolio/src/main/java/com/google/sps/data/Comment.java (hunk 1 above)
Comment (MrBanananator) [reply portfolio/src/main/java/com/google/sps/data/Comment.java]: Done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (MrBanananator) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Also done

**Pull request merged**